Modules/Loadable/Markups/MRMLDM/vtkMRMLMarkupsDisplayableManagerHelper.cxx

```diff
@@ -370,18 +383,23 @@ void vtkMRMLMarkupsDisplayableManagerHelper::AddWidget(vtkMRMLMarkupsDisplayNode
   // There should not be a widget for the new node
   if (this->GetWidget(markupsDisplayNode) != nullptr)
   {
-    vtkErrorMacro("vtkMRMLMarkupsDisplayableManager2D: A widget is already associated to this node");
+    vtkErrorMacro("vtkMRMLMarkupsDisplayableManagerHelper: A widget is already associated to this node");
     return;
   }
 
-  int wasModified = 0;
-  if (markupsDisplayNode->GetDisplayableNode())
+  vtkMRMLNode* markupsNode = markupsDisplayNode->GetDisplayableNode();
+  if (!markupsNode)
   {
-    // Prevent potential recursive calls during UpdateFromMRML call before the new widget is stored
-    // in MarkupsDisplayNodesToWidgets.
-    wasModified = markupsDisplayNode->GetDisplayableNode()->StartModify();
+    // No markups node associated with the display node.
+    // Nothing to display.
```

Comment:
Add something like you described in the commit message: "This is not an error, the widget will be created once both nodes are present"